backend/src/main/kotlin/app/ehrenamtskarte/backend/stores/importer/steps/RawToImportAcceptingStore.kt

```diff
@@ -23,6 +23,10 @@ class RawToImportAcceptingStore(private val logger: Logger) : PipelineStep<List<
                 it.discount,
                 it.category!!.toInt()
             )
+            if (it.houseNumber != null && it.houseNumber!!.isNotBlank())
+                store.copy(street = store.street?.trim() + " Nr. " + it.houseNumber)
```

Comment:
@aiquita Usually, you would say "Maximilianstraße 10", not "Maximilianstraße Nr. 10"
```suggestion
                store.copy(street = store.street?.trim() + " " + it.houseNumber)
```